### PR TITLE
Add async and use_ssl as a configuration options

### PR DESCRIPTION
**Pull request by indiebrain**
Summary
-------

Sometimes, in supporting rake tasks and the like, we want to explicitly configure the "options" of the InfluxDB client. The goal of this PR is to expose the `async` and `use_ssl` options as first-class configuration values of the `Ringflux` client while maintaining backwards compatibility with default values.

### CODE REVIEW

**Review by bklang — APPROVED** [review]

**Review by benlangfeld — APPROVED** [review]

**Comment by indiebrain** [comment]
Hey @benlangfeld and @bklang I've expanded the scope of this change-set - we needed to expose more than just the async option to achieve the correct behavior of the `Ringflux` client in our rake task. Would you mind reviewing this again?

In the event of a 👍 , I do _not_ have write access to this repo so I can't merge this into develop. Would one of y'all mind helping out with that? The current plan is to add these configuration values to the Ringflux client before merging https://github.com/powerhome/nitro-talkbox/pull/357 so fetch the `Ringflux` gem for the canonical source rather than my fork.

**Comment by benlangfeld** [comment]
@indiebrain @powerhome/cerebro now has write access to this repo.

**Review by benlangfeld — APPROVED** [review]

**Comment by benlangfeld** [comment]
@bklang I don't have owner on this gem. Would you add both me and Aaron?

`gem owner --add ben@langfeld.me ringflux`

**Comment by bklang** [comment]
@benlangfeld has been added as a Gem owner. @indiebrain let me know your Rubygems email/username and I'll add you there as well.

**Comment by indiebrain** [comment]
@bklang, I'm this guy: https://rubygems.org/profiles/indiebrain

**Comment by bklang** [comment]
@indiebrain done:

```
$ gem owner --add indiebrain ringflux
Owner added successfully.
Owners for gem: ringflux
- aaron.kuehler@gmail.com
- ben@langfeld.me
- bklang
```

**Comment by indiebrain** [comment]
Thanks y'all!

**Pull request merged**
